src/math/p5.Vector.js

```diff
@@ -1371,7 +1371,7 @@ p5.Vector.prototype.setMag = function setMag(n) {
 };
 
 /**
- * Calculate the angle of rotation for this vector (only 2D vectors)
+ * Calculate the angle of rotation for this vector in radians(only 2D vectors)
```

Comment:
line 1438 actually takes into consideration angleMode. so maybe you could phrase this is:
Calculate the angle of rotation for this vector. p5.Vectors created using `createVector()` will take the current angleMode into consideration, and give the angle in radians or degrees accordingly.

Reply:
I see, 
I'll update it as so then

Reply:
I've updated the inline docs, and added links to both createVecto() and angleMode() function